db/migrate/20160324174218_create_container_deployment_nodes.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContainerDeploymentNodes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_nodes do |t|
+      t.string :ip_or_hostname
+      t.string :name
+      t.text :labels
+      t.references :container_deployment
+      t.references :vm
```

Comment:
Change these to:

```ruby
t.belongs_to :container_deployment, :type => :bigint
t.belongs_to :vm,                   :type => :bigint
```

`t.belongs_to` for consistency, and `bigint` because all of our IDs are `bigint`s.